### PR TITLE
[EOSF-677] Google Analytics on job links

### PR DESCRIPTION
**Pull request by baylee-d**
## Ticket

https://openscience.atlassian.net/browse/EOSF-677

## Purpose

To add Google Analytic tracking to outbound links on the job page.

## Changes

- Added a function to the html to track outbound links that call it on click.  
- Changed the Job model to use a RawHTML field instead of a RichTextField.  This will let you directly alter the links in 'Apply' section of the job description.
- Added migrations to help change the data to JSON.  That way previous text shouldn't have to be changed.

## Additional

- This fix only changes the structure of the model in preparation for Google Analytics to be added to links.  In order for it to be added to links, someone with Admin access will have to change the links to have an `onclick` function like this:
` <a href="http://www.example.com" onclick="trackOutboundLink('http://www.example.com'); return false;">Check out example.com</a> `

This will make a call to the trackOutboundLink function and track the click in Google Analytics.

### CODE REVIEW

File: common/migrations/0054_auto_20170621_1404.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2017-06-21 14:04
```
Comment (jamescdavis) [inline common/migrations/0054_auto_20170621_1404.py]: Remove this since you've customized it.

**Review by jamescdavis — CHANGES_REQUESTED** [review]
Please rename common/migrations/0054_auto_20170621_1404.py to 0054_description_of_what_this_does.py to identify this as a custom migration.

**Comment by jamescdavis** [comment]
Should rebase and make sure migrations are ok.

**Comment by baylee-d** [comment]
Migrations should be all up to date and in order now 👍

**Review by jamescdavis — APPROVED** [review]
Migrations look good now.

**Pull request merged**